backend/src/equipment/equipment.service.ts

```diff
@@ -100,6 +121,19 @@ export class EquipmentService {
       });
       return EquipmentService.convertToDTO(equipment);
     } catch (error) {
+      if (error?.code === PostgresErrorCode.UniqueViolation) {
```

Comment:
used the `meta.target` property of the thrown `error` to identify what field unique field in the equipment is existing